_posts/2022-06-08-#22778.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: pr
+date: 2022-06-08
+title: "net processing: Reduce resource usage for inbound block-relay-only connections"
+pr: 22778
+authors: [jnewbery]
+components: ["p2p"]
+host: larryruane
+status: upcoming
+commit: 9db82f1
+---
+
+## Notes
+
+- Each of our node's peers is either inbound (our
+  peer initiated the connection) or outbound (we initiated the connection).
+  Once the connection is established, the relationship is symmetric (the
+  protocol works the same in both directions).
```

Comment:
To a certain extent, yes. The protocol is symmetric, but our treatment of nodes depends on whether the node is inbound or outbound. For example, we'll prefer fetch transactions or download blocks from outbound peers.

Reply:
The latest change made it sound like preferential download from outbound is the *only* difference, so I've edited the wording a little bit in 001b263